src/authorizations/components/TokenRow.tsx

```diff
@@ -42,6 +41,7 @@ class TokenRow extends PureComponent<Props> {
     return (
       <ResourceCard
         contextMenu={this.contextMenu}
+        contextMenuInteraction="alwaysVisible"
```

Comment:
i feel like this string `"alwaysVisible"` should be made into an enum so we get some type safety and consistency. what do you think?

Reply:
It's typed as allowing specific strings to be passed in, which is somewhere between an enum and any string:
https://github.com/influxdata/clockface/blob/master/src/Components/ResourceList/Card/ResourceCard.tsx#L20-L21

Changing it seems out of scope for this PR, but def worth considering